changelog/README.md

```diff
@@ -0,0 +1,34 @@
+How to add a new changelog entry to the pending changelog?
+==========================================================
+
+This will get copied to dlang.org and cleared when master gets
+merged into stable prior to a new release.
+
+```
```

Comment:
> Please see dlang/phobos#4228 for the motivation and examples.

Sorry, the intention was not confusing, but more how should I use it :-). As being one that wasn't involved in the motivation/examples, I just wanted to say that the `README` is confusing as it doesn't provide clear guidelines what one should do (and there's no prior art in the same repo to rip it off)